### DISCUSSION TITLE
PhysX error: Illegal BroadPhaseUpdateData

### DISCUSSION THREAD

**Discussion by JianlinWang-JL** [isaac-sim/IsaacLab#1095] [Q&A]
### Question
Hi, I am trying to run a self-designed robot with Orbit Framewok, and I keep having the following questions:

![image](https://github.com/NVIDIA-Omniverse/Orbit/assets/133347976/1a290a7c-f365-45c6-8493-62fb0c820dc6)

And shown in the figure, it says, Invalid PhysX transform detected for "/World/Robot_1/logRobotDual/F_L_Lifting", which is every coordinate in the USD, as shown in the following figure:
![image](https://github.com/NVIDIA-Omniverse/Orbit/assets/133347976/350f3f39-31cb-4f80-8bf7-967bc4e2b830)

I have no idea which part is wrong and many thanks to any advise and help!

**Comment by JianlinWang-JL** [comment]
Many thanks to any replies and happy to discuss more about it!

**Comment by Mayankm96** [comment]
Hi, can you please provide more information on what you are executing? These errors normally happen if some transform is being set explicitly on an articulation. Articulations in PhysX only allow setting their state using root and joint states.

**Comment by JianlinWang-JL** [comment]
Hi @Mayankm96, Thank you so much for your reply! I am trying to import a self-designed robot into the environment based on orbit, for now there are still no actions applied to it. Still have no idea what is going on.

**Comment by Dhoeller19** [comment]
Hi @JianlinWang-JL, seems like the set-up of your articulation in USD is wrong, which leads to physx exploding.
